src/Kafka/KafkaTopic/types.ts

```diff
@@ -0,0 +1,73 @@
+import { RetentionSizeUnits, RetentionTimeUnits } from "../types";
+
+export type KafkaTopic = {
+  topic_name: string;
+  partitions: number;
+  retention_time: string;
+  retention_size: string;
+};
+
+export type KafkaTopicField = keyof KafkaTopic;
+
+export const unitsToBytes = {
+  [RetentionSizeUnits.BYTE]: 1,
+  [RetentionSizeUnits.KIBIBYTE]: 1024,
+  [RetentionSizeUnits.MEBIBYTE]: 1048576,
+  [RetentionSizeUnits.GIBIBYTE]: 1073741824,
+  [RetentionSizeUnits.TEBIBYTE]: 1.0995116e12,
+};
```

Comment:
For the metrics we use an [external lib](https://github.com/redhat-developer/app-services-ui-components/blob/b97ccae1133e0ccec1c1f2ece120e8aaf8f441af/src/Kafka/Metrics/components/utils.ts#L7-L9) to convert bytes. In general, for solved problems like these, I'd prefer relying on something external (end tested) than writing some custom code ourselves.

No need to address this in this PR, but I'd make a note to streamline all these conversions. It's clear how it's a shared concern across components, so we should have some shared component/utility to deal with it.

Reply:
yeah sure, will make use of the external lib